lib/utils/error-messages.js

```diff
@@ -593,4 +609,56 @@ function getEventBasedGatewayTargetNotAllowedErrorMessage(report) {
   }
 
   return report.message;
+}
+
+function getPropertyValueNotAllowedErrorMessage(report, executionPlatform, executionPlatformVersion, modeler = 'desktop') {
+  const {
+    data,
+    message
+  } = report;
+
+  const {
+    allowedVersion,
+    node,
+    parentNode,
+    property
+  } = data;
+
+  const typeString = getTypeString(parentNode || node);
+
+  if (is(node, 'zeebe:CalledElement') && property === 'propagateAllParentVariables') {
+    return getSupportedMessage(`${ getIndefiniteArticle(typeString) } <${ typeString }> with <Propagate all variables> disabled`, executionPlatform, executionPlatformVersion, allowedVersion);
+  }
+
+  return message;
+}
+
+function getSecretExpressionInvalidErrorMessage(report) {
+  const { data } = report;
+
+  const { property } = data;
+
+  return `Property <${ property }> is not a valid secret`;
+}
+
+function getElementPropertyValueDuplicatedErrorMessage(report) {
+  const {
+    data,
+    message
+  } = report;
+
+  const {
+    node,
+    parentNode,
+    duplicatedProperty,
+    duplicatedPropertyValue
+  } = data;
+
+  const typeString = getTypeString(parentNode || node);
+
+  if (is(node, 'bpmn:LinkEventDefinition') && duplicatedProperty === 'name') {
+    return `${ getIndefiniteArticle(typeString) } <${ typeString }> with the same <Name> (${ duplicatedPropertyValue }) as another is not supported`;
```

Comment:
I find the error message a bit confusing. What do we think about something like

> Event_1: A \<Link Intermediate Catch Event\> must have a unique \<Name\> (messageName)

```suggestion
    return `${ getIndefiniteArticle(typeString) } <${ typeString }> must have a unique <Name> (${ duplicatedPropertyValue })`;
```

Reply:
I like. 👍🏻